### ISSUE TITLE
utilize the nushell fill command

### ISSUE THREAD

**Issue by fdncred** [glcraft/nu-scripts#2]
I was looking at the code where you're drawing ------- SOME TEXT -------.  You might be able to simplify some of your code and do something like this. It's meant to align text.
```
$'(ansi default_reverse)nushell(ansi reset)' | fill -a m -c '─' -w 1
```

**Comment by glcraft** [comment]
Oh you're right. I was searching something like `str fill` or `repeat` but didn't find
Thanks 👍

**Comment by glcraft** [comment]
Fixed in ea7811e19f16a2377b6632ab0c97dadb320ec635 👍
